public/lib/inverted-index-spec.js

```diff
@@ -0,0 +1,193 @@
+// const values = require('object.values');
+
+//  book with valid contents
+const InvertedIndexClass = require('../../app/class/invertedIndexClass');
```

Comment:
I'm curious, how would this work?
I was expecting to see export in your InvertedIndexClass before this would work